### PR TITLE
Updating markup to display new user experience across the board

### PR DESCRIPTION
**Pull request by kenperkins**
This should get the site up to date with the look and feel that has been flighted on VWO.

### CODE REVIEW

**Comment by rgbkrk** [comment]
Can we :shipit:?

**Comment by kenperkins** [comment]
Someone should CR @rgbkrk

File: src/site_source/_includes/footer.html [hunk 1]
```diff
@@ -1,33 +1,19 @@
...
-					</span> |
-					<span class="footerlink">
-						<a href="http://www.rackspace.com/information/legal/websiteterms" class="basement">Website Terms</a>
-					</span> |
-					<span class="footerlink">
-						<a href="http://www.rackspace.com/information/legal/copyrights_trademarks" class="basement">Trademarks</a>
-					</span>
-        </div>
-      </div>
+        <p>Questions? Read the <a href="#">FAQs</a>. | Found a bug or typo? <a href="#">Report it.</a></p>
```
Comment (rgbkrk) [inline src/site_source/_includes/footer.html]: Is JavaScript handling the links here?

File: src/site_source/_includes/footer.html [hunk 2]
```diff
@@ -1,33 +1,19 @@
...
     </div>
-  </div>
-
 </div>
+<div class="basement">
+    <div class="container">
+        <img src="{% asset_path rackerpowered-logo.png %}" alt="Racker Powered">
+        <ul>
+            <li>©2014 Rackspace, US Inc.</li>
```
Comment (rgbkrk) [inline src/site_source/_includes/footer.html]: Might as well say 2015 now.

File: src/site_source/index.html [hunk 3]
```diff
@@ -3,82 +3,135 @@
...
 
-    <div class="tribox col-md-4 col-sm-4">
-      <div class="inner">
-        <img src="{% asset_path sdks-icon.svg %}" alt="SDKs and Tools">
-        <span class="boxtitle">SDKs &amp; Tools</span>
+        <div class="row">
+            <div class="col-md-6">
+                <div class="row">
+                    <div class="col-md-4">
+                        <div class="new-landing-sprite sdk-graphic" alt="SDKs & tools"></div>
```
Comment (rgbkrk) [inline src/site_source/index.html]: Does this `&` need escaped `&amp;`?

File: src/site_source/index.html [hunk 4]
```diff
@@ -3,82 +3,135 @@
...
-      <hr/>
+<div class="box quote">
+    <div class="container">
+        <blockquote>
+            <span class="quotes">“</span>We built developer+<sup>TM</sup> (a monthly infrastructure credit and
+            more) to give developers the core of almost any scalable
+            cloud application they want to build.<span class="quotes">”</span>
+        </blockquote>
+        <div class="author">Jesse Noller</div>
+        <div class="author-title">Former Python Software Foundation board member, Principle Engineer & Senior Manager
```
Comment (rgbkrk) [inline src/site_source/index.html]: Another escapee. `&amp;`

File: src/site_source/index.html (hunk 3 above)
Comment (rgbkrk) [inline src/site_source/index.html]: Optionally, maybe refactor `new-landing-sprite` to be `landing-sprite` now?

File: src/site_source/_includes/footer.html (hunk 1 above)
Comment (kenperkins) [inline src/site_source/_includes/footer.html]: Good catch. No.

**Comment by rgbkrk** [comment]
Other than small tweaks, I think this is great. I'd like to ship as soon as possible since this is a hefty PR (in line count).

File: src/site_source/index.html (hunk 3 above)
Comment (kenperkins) [inline src/site_source/index.html]: we're going to be re-doing this soon anyway. It's good enough?

**Comment by kenperkins** [comment]
Thanks @rgbkrk addressed.

**Pull request merged**